frontend/src/components/ResultsTable.js

```diff
@@ -13,8 +15,15 @@ class ResultsTable extends Component {
         switch (typeof v) {
           case 'boolean':
             return v.toString();
+          case 'string':
+            // assume any multiline string is code
+            if (v.split('\n').length > 1) {
```

Comment:
Would this in JS work for both platform-dependent line endings?

Would it be possible to include a simple test for this function, so further refactoring would rely on it?

On the side-note, in case >= 1 could work for this, may be some kind of equivalent of `.indeOf > 0` could be more performant..

Reply:
window line ending is `\r\n` so it will work.
👍 for `indexOf`.